### PR TITLE
 ES 2.x: Fixing metric types for es 2 new metrics

### PR DESCRIPTION
**Pull request by vzhabiuks**

### CODE REVIEW

**Comment by vzhabiuks** [comment]
@mdubbyap , @mahdibh  Could you review?

File: elasticsearch_collectd.py [hunk 1]
```diff
@@ -211,13 +211,13 @@
...
         Stat("gauge", "nodes.%s.indices.query_cache.cache_size"),
     'indices.cache.filter.hit-count':
-            Stat("gauge", "nodes.%s.indices.query_cache.hit_count"),
+            Stat("counter", "nodes.%s.indices.query_cache.hit_count"),
     'indices.cache.filter.miss-count':
-            Stat("gauge", "nodes.%s.indices.query_cache.miss_count"),
+            Stat("counter", "nodes.%s.indices.query_cache.miss_count"),
     'indices.cache.filter.cache-count':
-            Stat("gauge", "nodes.%s.indices.query_cache.cache_count"),
+            Stat("counter", "nodes.%s.indices.query_cache.cache_count"),
```
Comment (mahdibh) [inline elasticsearch_collectd.py:218]: This seems like a gauge. Can you double check ?

Comment (eabbott) [inline elasticsearch_collectd.py:218]: Looks like a counter
https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/index/cache/query/QueryCacheStats.java#L91

```
    /**
     * The number of {@link DocIdSet}s that have been cached.
     */
    public long getCacheCount() {
        return cacheCount;
    }

    /**
     * The number of {@link DocIdSet}s that are in the cache.
     */
    public long getCacheSize() {
        return cacheSize;
    }
```

**Comment by mahdibh** [comment]
thanks @eabbott  !

**Comment by eabbott** [comment]
:+1:

**Pull request merged**